extensions/applicationinsights-properties-js/Tests/Unit/src/properties.tests.ts

```diff
@@ -962,7 +1180,6 @@ export class PropertiesTests extends AITestClass {
 
     private getTelemetryConfig(): IPropertiesConfig {
         return {
-            instrumentationKey: "",
```

Comment:
This should never have been on the `IPropertiesConfig` as it's never used or referenced from here, it's only ever used from the root (`IConfiguration`) definition -- So the removal is just removing dead code (defaults) and definitions.